src/style/shared/colors.less

```diff
@@ -95,6 +95,14 @@
 @search-text: #8e8e8e;
 @search-bright-text: #cacaca;
 
+/* Libraries */
+@libraries-dialog-dark-gray: #282828;
```

Comment:
Yes, some of these are repeats, it would be great to consolidate those definitions into 1 variable with a better name